test/TemplatesApiIntegrationTests.js

```diff
@@ -0,0 +1,37 @@
+const docusign = require('../src/index');
+const assert = require('assert');
+const { JWTAuth } = require('./helpers');
+const { apiClient } = require('./constants');
+
+let ACCOUNT_ID = '';
+
+describe('TemplatesApi tests:', () => {
+  before((done) => {
+    try {
+      JWTAuth(done).then((response) => {
+        ACCOUNT_ID = response.ACCOUNT_ID;
+        done();
+      });
+    } catch (err) {
```

Comment:
This is incorrect error handling for an async call and will cause fails to timeout and never call done(). Please use the .catch method, or preferably `await` depending on where we land on version deprecation.